rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    bool.rental << self
```

Comment:
- [ ] There's a typo, Please change `bool` to `book`.

Reply:
Oh wow, Thank you a lot